sdk/azidentity/bearer_token_policy_test.go

```diff
@@ -22,9 +22,9 @@ const (
 )
 
 func defaultTestPipeline(srv azcore.Transport, cred azcore.Credential, scope string) azcore.Pipeline {
-	retryOpts := azcore.DefaultRetryOptions()
-	retryOpts.MaxRetryDelay = 500 * time.Millisecond
-	retryOpts.RetryDelay = 50 * time.Millisecond
+	retryOpts := azcore.RetryOptions{
+		TryTimeout: 1 * time.Minute,
```

Comment:
Looks like the original values for `MaxRetryDelay` and `RetryDelay` were removed, was that intentional?

Reply:
Done..added them back in